docker-compose.yaml

```diff
@@ -3,61 +3,95 @@ version: '3.7'
 services:
 
   caddy:
-    container_name: caddy
-    image: abiosoft/caddy:1.0.3-no-stats
+    image: lucaslorentz/caddy-docker-proxy:latest
```

Comment:
Be careful when using latest tag. It is preferable to use a specific version, as latest tag might get breaking changes.
Latest now is caddy v2, which is not compatible with caddy v1.